### PR TITLE
Added claim taxes calculation on total skill assets

### PR DESCRIPTION
**Pull request by rodipm**
Added a new switch inside "Total Skill Assets" menu to include claim taxes discounts.

When claim taxes are included "Total Skill Assets" modal changes to "Taxed Skill Assets" and claim taxes discounts are calculated.

Tax calculations are based on "getOwnRewardsClaimTax" from cryptoblades contract.

Not sure if this is the best UI solution for this, but hope it helps.

### CODE REVIEW

File: public/js/core.js [hunk 1]
```diff
@@ -366,6 +380,10 @@ function fromEther (value) {
     return web3.utils.fromWei(BigInt(value).toString(), 'ether')
 }
 
+function convertClaimTax(value) {
+    return value*0.15/2767011611056432742
```
Comment (ed3ath) [inline public/js/core.js:384]: where does 2767011611056432742 come from? can you send a ref here? thanks

Comment (rodipm) [reply public/js/core.js:384]: Came from cryptoblades contract [(here)](https://bscscan.com/address/0x9c29136bc1617b011e6c125b616fbaba1d311b6e#code).

Inside "File 4"'s contract:
image

```
int128 public constant REWARDS_CLAIM_TAX_MAX = 2767011611056432742; // = ~0.15 = ~15%
```

Comment (rodipm) [reply public/js/core.js]: Also just noticed "REWARDS_CLAIM_TAX_MAX" function to retrieve the maximum claim tax value (2767011611056432742).

Just commited another version getting this value on document ready.

Comment (ed3ath) [reply public/js/core.js]: so that's the max tax? what about the daily -1%? where did you apply it?

Comment (ed3ath) [reply public/js/core.js]: or the contract itself subtract it automatically?

Comment (rodipm) [reply public/js/core.js]: Exactly,"getOwnRewardsClaimTax" function returns the updated value, that can be compared against the max value in order to retrieve current tax value in %.

Comment (ed3ath) [reply public/js/core.js]: i see

**Pull request merged**